gandalf.js

```diff
@@ -4,64 +4,73 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?